<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastrando</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(253, 254, 232);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="enviar_dados">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>360</y>
      <width>102</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 233, 19);
border-radius: 15px;
border-style: outset;
border-width: 0px;</string>
    </property>
    <property name="text">
     <string>Cadastrar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dado_nome">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>110</y>
      <width>242</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset;
border-width: 2px;</string>
    </property>
    <property name="placeholderText">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QLabel" name="msg_for_user">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>320</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(114, 75, 17);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirma_senha">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>270</y>
      <width>242</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset;
border-width: 2px;</string>
    </property>
    <property name="placeholderText">
     <string>Confirme sua senha</string>
    </property>
   </widget>
   <widget class="QLabel" name="imagem_dados">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>110</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Dados.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="by">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>400</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>by antthLuca</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_voltar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 233, 19);
border-radius: 15px;
border-style: outset;
border-width: 0px;</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dado_sobrenome">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>150</y>
      <width>242</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset;
border-width: 2px;</string>
    </property>
    <property name="placeholderText">
     <string>Sobrenome</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="login">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>190</y>
      <width>242</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset;
border-width: 2px;</string>
    </property>
    <property name="placeholderText">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="imagem_login">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>190</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Parte 1/Perfil.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="imagem_senha">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>230</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Parte 1/Chave.png</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="senha">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>230</y>
      <width>242</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset;
border-width: 2px;</string>
    </property>
    <property name="placeholderText">
     <string>Senha</string>
    </property>
   </widget>
   <widget class="QLabel" name="texto_botaoVoltar">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>44</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
   <widget class="QLabel" name="instrucao">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>60</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Preencha os campos abaixo para cadastrar um novo usu√°rio:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>484</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
